src/SignalR/common/Http.Connections/src/ConnectionOptionsSetup.cs

```diff
@@ -14,7 +14,7 @@ public class ConnectionOptionsSetup : IConfigureOptions<ConnectionOptions>
         /// <summary>
         /// Default timeout value for disconnecting idle connections.
         /// </summary>
-        public static TimeSpan DefaultDisconectTimeout = TimeSpan.FromSeconds(15);
+        public static TimeSpan DefaultDisconnectTimeout = TimeSpan.FromSeconds(15);
```

Comment:
We've previously discussed changing this and decided it's not worth making a breaking change here.
https://github.com/dotnet/aspnetcore/pull/18494